cleanslate/doma/templates/chore_delete_form.html

```diff
@@ -1,7 +1,12 @@
 {% extends 'base.html' %}
+{% block extra_head %}
+  {% load static %}
+  <link href="{% static 'css/form.css' %}" rel="stylesheet">
+{% endblock %}
 
-{% block content %}
 
+{% block content %}
+<div class = "card z-depth-5 center">
 <h1>Delete Chore</h1>
 
 <p>Are you sure you want to delete this chore?</p>
```

Comment:
The buttons on this and the edit page have no styling, perhaps standardize them like the create page?